helm/values-eks.yaml

```diff
@@ -172,11 +172,13 @@ common:
       kubernetes.io/ingress.class: alb
       alb.ingress.kubernetes.io/tags: service_instance=production
       alb.ingress.kubernetes.io/scheme: internet-facing
-      alb.ingress.kubernetes.io/certificate-arn: arn:aws:acm:us-east-2:832573439839:certificate/a7db79d4-4f8c-4c7b-a4f4-109bf2fb5efa
+      alb.ingress.kubernetes.io/certificate-arn: arn:aws:acm:us-east-2:832573439839:certificate/577f9991-a6cf-4d1c-b56d-447b24077f46
```

Comment:
This is just the arn? Can we as some comments here? As this converts to docs

Reply:
Sure. Done